policyengine_us/variables/gov/states/va/tax/income/va_agi.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class va_agi(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Virginia Adjusted Gross Income (VAGI)"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://law.lis.virginia.gov/vacodefull/title58.1/chapter3/article2/"
+    )
+    defined_for = StateCode.VA
+
+    adds = [
+        "adjusted_gross_income",
```

Comment:
include the va_age_deduction in the calculation 
include the taxable_social_security in the calc

Reply:
i don't understand these. federal AGI already includes SS, and why would a deduction be an addition?

Reply:
flipped, should be subtractions